.github/auto_request_review.yml

```diff
@@ -0,0 +1,22 @@
+# More info at https://github.com/necojackarc/auto-request-review
+reviewers:
```

Comment:
out of curiosity what is the difference between this a code owners? 

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Reply:
the difference comes into play with the branch protection rules. 

one of the branch protection rules is that to require Code Owner approval, which is a good setting to have enabled.

However if we are using the code owners to auto assign PRs for the entire team (which is basically what we are doing now), it then makes the fact that we have code owners a moot point. 

This allows use to have specific people on the team review specific things, while maintain a select few code owners.